disnake/state.py

```diff
@@ -1956,17 +1956,17 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         self.shard_ids: Union[List[int], range] = []
         self.shards_launched: asyncio.Event = asyncio.Event()
 
-    def _update_message_references(self) -> None:
-        # self._messages won't be None when this is called
-        for msg in self._messages:  # type: ignore
+    def _update_guild_channel_references(self) -> None:
+        messages: Sequence[Message] = self._messages or []
+        for msg in messages:
```

Comment:
this is minor, but let's be a bit more clear:
```suggestion
        if not self._messages:
            return
        for msg in self._messages:
```

Reply:
sounds good, changed in 165907e1de83f11e952d8dddea7feea120d10740